src/components/common/FileUploadPdf.vue

```diff
@@ -0,0 +1,316 @@
+<template>
```

Comment:
We should 100% move this to BCRS-SHARED-COMPONENTS

Reply:
Go ahead and create a new ticket.

Reply:
https://github.com/bcgov/entity/issues/16489

Reply:
New ticket is bcgov/entity#16489.